gantry/firmware/interfaces.cpp

```diff
@@ -105,6 +105,7 @@ void interfaces::initialize() {
     if (initialize_spi(get_axis_type()) != HAL_OK) {
         Error_Handler();
     }
+    interfaces::get_motor().driver.setup();
```

Comment:
can we put this in another pr

Reply:
This looks like it is from another PR. Perhaps this branch should be rebased off main?